### DISCUSSION TITLE
Server-side rendering support

### DISCUSSION THREAD

**Discussion by Dr-AkA** [dotnet/maui#9832] [Ideas]
There is so important service which is not included in maui, we need server side rendering.

**Comment by hartez** [comment]
What do you mean by "server side rendering"?

**Comment by Dr-AkA** [comment]
> What do you mean by "server side rendering"?

I want to host my blazor component in server side when i update the ui in server my client app updates the ui too without updating actuall app in the app store, i used hybrid approach but i need native features in my client app like accessing gps,printing

**Comment by janseris** [comment]
What about WebView with registered protocol URL to be recognized by the app?

**Comment by Dr-AkA** [comment]
> What about WebView with registered protocol URL to be recognized by the app?

The problem is if i do webview i lost native functions which is important for my app, if i hit a print button nothing happens on phone otherwise in the browser it works

**Comment by Dr-AkA** [comment]
> What about WebView with registered protocol URL to be recognized by the app?

The problem is if i do webview i lost native functions which is important for my app, if i hit a print button nothing happens on phone otherwise in the browser it works

**Comment by janseris** [comment]
>with registered protocol URL to be recognized by the app

**Comment by Dr-AkA** [comment]
> > with registered protocol URL to be recognized by the app

how can you explain it more please

**Comment by janseris** [comment]
https://stackoverflow.com/questions/18102528/android-how-to-create-custom-url-scheme-with-the-given-format-myapp-http

**Comment by Dr-AkA** [comment]
> https://stackoverflow.com/questions/18102528/android-how-to-create-custom-url-scheme-with-the-given-format-myapp-http

Bro i did this but i want to access native functions of phone like gps and accelerometer

**Comment by janseris** [comment]
You will have to pass the result from the app in the device (which is invoked from the web app) back to your server.

**Comment by Dr-AkA** [comment]
Bro, i made maui app runs on server now i want to make this app runs on
android and ios, when i update the ui in server the phone apps fetches the
update automatically and i need to access the native function of devices as
well, when i press print button the android do the printing for me or ios
do that.
